src/containers/register/reducer.js

```diff
@@ -0,0 +1,25 @@
+import { REGISTER } from './constants'
+
+const initialState = {
```

Comment:
Use an objet, not an array. You will need to handle errors coming from the API, I suggest you to use the state's shape:
```javascript
{
  success: false,
  error: '',
}
```

For `_SUCCESS` action update `success` to `true` (and so show the info message in the component). For `_ERROR`, put the error into `error` and display it in the component.